integration_test/third_party_apps_data/applications/solr/centos_rhel/install

```diff
@@ -0,0 +1,40 @@
+set -e
+
+sudo setenforce 0
+
+cat <<EOF | sudo tee /etc/selinux/config
+SELINUX=disabled
+SELINUXTYPE=targeted
+EOF
+
+sudo yum install -y \
+    java-1.8.0-openjdk java-1.8.0-openjdk-devel curl lsof
+
+curl -L -o \
+    solr-8.11.1.tgz \
+    https://downloads.apache.org/lucene/solr/8.11.1/solr-8.11.1.tgz
```

Comment:
Is it possible to use regular yum/apt-get/zypper install?

Chances are a bit higher that users will use the default package manager (e.g. yum, apt-get, zyypper) to install a package instead of a pinned version of the application. The install (aka integration_test/third_party_apps_data/applications/{APP}/{DISTRO}/install) script should try reflect the native installation methods as much as possible.

Same for the other `install` scripts.

Reply:
@qingling128 it doesn't look like this option is available for solr. their official install instructions just show using a tarball

Reply:
Debian 9 and 10 appear to have packages for solr: https://packages.debian.org/buster/solr-tomcat
as do Ubuntu focal and bionic: https://packages.ubuntu.com/focal/solr-tomcat

Reply:
@quentinmit I don't believe those are what we want to use (and conferred with our dev ops team about this) for a base Solr installation. We are following the instructions of what is in the official install doc https://solr.apache.org/guide/7_0/installing-solr.html

Reply:
Debian's docs say that `apt install solr-tomcat` will give you a running Solr with zero configuration: https://wiki.debian.org/Solr

I'll leave it up to @qingling128 what we want to do here. I'd love if we could test both installation methods - the most important part of these integration tests is testing distro-specific details, since otherwise we could just do all the tests on a single distro with manually-installed software. So maybe use the packages on the Debian and Ubuntu releases that have them, and the tarball on the other distros, so we cover them both.

Reply:
@quentinmit so I tried out that package, but the final installation state was very different than when using the tarball. Also, the documentation that you linked to (https://wiki.debian.org/Solr) refers to features like solr.xml which was deprecated in 2014 and unsupported in 2015. I think it's installing a pretty old version of Solr.

Reply:
Started a side conversation around `Test Ops Agent Supported Application Versions`. Shared the doc. Feel free to chime in there. I think we can keep discussing this separately (aka not a blocker for this PR).